### PR TITLE
fix(labs): correct debt multiplier answer in lab 14 Part D

### PR DESCRIPTION
**Pull request by Shashank-Tripathi-07**
## Summary

Part D asks: "You defer retraining for 3 consecutive T* cycles. Total accuracy loss vs single miss?"

The answer key marked **"C) ~5-6x (debt compounds)"** as correct, but the debt cascade formula with default sliders (N=3 missed cycles, 2 downstream models, 2 pp/cycle, alpha=1.3) computes:

```
compound loss = sum(2 * k^1.3, k=1..3)  = 2*(1 + 2.46 + 4.73) = 16.4 pp
cascade cost  = 3 * 2 * 2.0 * 0.3       =                        3.6 pp
total                                    =                       20.0 pp
multiplier    = 20.0 / 2.0              =                        ~10x
```

The dynamic display card already showed `~10x` while the answer marked `~5-6x` as correct -- a visible contradiction.

## Changes

All four places that stated the wrong "5-6x" figure are corrected to "~10x":
- Learning objective text
- Radio option C label: `"~5-6x (debt compounds)"` → `"~10x (debt compounds)"`
- Answer key value: `"5x"` → `"10x"`
- Synthesis summary callout

## Test plan

- [ ] Open lab 14 Part D with default sliders (3 cycles, 2 downstream, 2 pp/cycle)
- [ ] Confirm debt multiplier card shows ~10x
- [ ] Select "C) ~10x (debt compounds)" -- confirm "Correct" callout appears

### CODE REVIEW

**Comment by profvjreddi** [comment]
Thanks @Shashank-Tripathi-07! 🎉

@all-contributors please add @Shashank-Tripathi-07 for bug, code in labs
